CONTRIBUTING.md

```diff
@@ -54,26 +54,22 @@ code changes and the commit message should start with `Docs:`.
 
 ### Run the docs locally
 
-Make docs changes and test them by using hugo to run the kpt site locally. Hugo
-continuously builds your website as you make changes.
-
-- install hugo
-- `make servedocs`
+Make docs changes and test them by running the site in a docker container
+with `make site-run-server`.
 
 It's usually a good idea to test locally for the following:
 
 - Broken links
 - Rendering static content
-- Ordering of new pages using the "weight" key in the header
 
 ### Update docs
 
-Docs are under [site/] and use the [docsy] theme for hugo. Learn more about
-docsy from [docsy docs]. The site itself are generated from the markdown
-automatically after commit.
+Docs are under [site/] and use [docsify] to present the source markdown files.
```

Comment:
Comments about the lines below:

- [ ] `make lintdocs`: this doesn't work.
- [ ] Let's remove the section about Omnigraffle. We're going to remove all these images because they're not readable since they pack too much in one image, and they're also stale.